python-regression/tests/features/machine2/2_transaction_tests.feature

```diff
@@ -87,3 +87,35 @@ Feature: Test transaction confirmation
             | keys   | values | type          |
             | states | False  | boolListMixed |
 
+
+    Scenario: ZMQ receives transaction streams
+        Sends a predefined transaction object and a milestone that references it. This should trigger a series
+        of zmq stream publications. The responses for these streams are then checked against the expected contents of
+        the stream.
+
+        #Subscribe to zmq stream transaction topics
+        Given "nodeA-m2" is subscribed to the following zmq topics:
+        | keys                  |
+        | sn                    |
+        | sn_trytes             |
+        | tx                    |
+        | tx_trytes             |
+
+        Then "storeTransactions" is called on "nodeA-m2" with:
+        |keys                   |values                                 |type           |
+        |trytes                 |TRANSACTION_TEST_TRANSACTION_TRYTES    |staticList     |
```

Comment:
Why did you make new trytes/static vals here? Couldnt we just reuse TEST_TRYTES/EMPTY_TRYTES?